examples/electron/src/renderer/pages/user/ui/user.tsx

```diff
@@ -0,0 +1,27 @@
+import { ChangeEvent, useState } from 'react';
+import { saveUser } from '@/pages/user/ipc/save-user';
```

Comment:
issue: внутри одного слайса FSD не рекомендует делать абсолютные импорты, чтоб не путаться, и чтоб случайно не импортировать из индекса, создав циклический импорт

suggestion:
```suggestion
import { saveUser } from '../ipc/save-user';
```